_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -14,7 +14,11 @@ input_aspects:
   - DOM Tree
   - CSS Styling
 authors:
-  - Anne Thyme Nørregaard
+	- Anne Thyme Nørregaard
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint 
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
```

Comment:
Are those SVGs supposed to have an `html` tag on the code snippet? Otherwise if you put `svg` on them they'll be ignored, and this won't be an issue.